src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/InputInProcess.cs

```diff
@@ -32,6 +41,14 @@ internal async Task SendAsync(params object[] keys)
             SendKeys.Send(keys);
         }
 
+        internal async Task SendToNavigateToAsync(params object[] keys)
```

Comment:
📝 This is super-icky. I want to rewrite `SendKeys` implementation after old integration tests no longer need it.